pages/_app.js

```diff
@@ -25,7 +25,7 @@ function InternalApp({ Component, pageProps: { ...pageProps } }) {
   const [isDarkMode, setIsDarkMode] = useLocalStorageState("darkTheme", {
     defaultValue: false,
   });
-  const theme = isDarkMode ? darkTheme : lightTheme;
+  const themeRootVariablesFunc = isDarkMode ? darkTheme : lightTheme;
```

Comment:
How about something like `themeToggle`? 🤔

Reply:
The variable has the same name as the theme property 🙂